addons/hr_work_entry_holidays/tests/test_performance.py

```diff
@@ -41,7 +41,7 @@ def test_performance_leave_validate(self):
     def test_performance_leave_write(self):
         leave = self.create_leave(datetime(2018, 1, 1, 7, 0), datetime(2018, 1, 1, 18, 0))
 
-        with self.assertQueryCount(__system__=19, admin=27):
+        with self.assertQueryCount(__system__=24, admin=30):
```

Comment:
here as well.. I guess because we do extra check on write and create? but still, would be good to know why the increase